hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -410,6 +410,16 @@
       hwext: "true",
       hwqe: "true",
       fields: [
+        { bits: "14",
+          name: "CLKIDLE",
+          desc: '''Suppress output clock edges for this command.
+
+                   This keeps SCK at the idle value throughout this command.
+                   As a result, different patterns can be created on the pins without clocking data.
+                   This could be used for software control of the chip select, for example.
```

Comment:
Here's how:

To set CSB, set CLKIDLE and CSAAT to 1, and set DIRECTION to 0.
To lift CSB, set CLKIDLE to 1, and set CSAAT and DIRECTION are 0.